content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -99,13 +99,13 @@ We allow to configure:
   * Check [Network Policy documentation](https://kubernetes.io/docs/concepts/services-networking/network-policies/#networkpolicy-resource) to see available options in the spec.
   * Also check a [common services connectivity issue](#i-created-a-load-balancer-service-on-a-user-cluster-but-services-outside-cannot-reach-it) that can be solved by a custom network policy.
 * `dnsConfig` and `dnsPolicy` - DNS config and policy which are set up on a guest. Defaults to `ClusterFirst`.
-  * You should set those fields when you suffer from DNS loop or collision issue. [Refer to this section for more details.](#i-discovered-a-dns-collision-on-my-cluster-why-it-happens) 
+  * You should set those fields when you suffer from DNS loop or collision issue. [Refer to this section for more details.](#i-discovered-a-dns-collision-on-my-cluster-why-does-it-happen) 
 * `images` - Images for Virtual Machines that are selectable from KKP dashboard.
   * Set this field to make sure that users can select operating systems for their VMs. 
 * `infraStorageClasses` - Storage classes that are initialized on user clusters that end users can work with.
   * Pass names of KubeVirt storage classes that can be used from user clusters.
 
-Refer to this [document](https://github.com/kubermatic/kubermatic/blob/main/docs/zz_generated.seed.ce.yaml#L116)
+Refer to this [document](https://github.com/kubermatic/kubermatic/blob/main/docs/zz_generated.seed.ce.yaml#L115)
```

Comment:
The generated `Seed` is also available here: https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/project-and-cluster-management/seed-cluster/. Should maybe link to that page instead of GitHub?

Reply:
Yeah probably, although to be honest i hate that page as you cannot point to the specific part, you just have a big pile of yaml.

Reply:
done

Reply:
You could also link to the CRD docs though. that would allow you to at least link to the specific sub-type.